static/images/help-purple.svg

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<svg
```

Comment:
When following how other SVG terms are added, it's _before_ the `<xml>` tag. You may swap these two lines' order. 

Firefox SVG reference: https://searchfox.org/mozilla-central/source/toolkit/themes/shared/icons/arrow-down-12.svg